dlnameta.c

```diff
@@ -374,15 +380,15 @@ get_dlna_metadata_video_ctx(struct AVFormatContext *ctx, int audio_stream, int v
 {
 	struct dlna_meta_s m = {0, 0};
 
-	struct AVCodecContext *ac = NULL;
-	struct AVCodecContext *vc = NULL;
+	struct AVCodecParameters *ac = NULL;
+	struct AVCodecParameters *vc = NULL;
```

Comment:
When compiling this, I get a warning. `vc` is being passed to [lav_get_interlaced](https://github.com/JuanKman94/readymedia-transcode/blob/transcode/libav.h#L158). I tried replacing that parameter to `AVCodecParameters` but then I get a bunch of other warnings as [AVCodecParameters](https://libav.org/documentation/doxygen/master/structAVCodecParameters.html) does not have the [time_base](https://libav.org/documentation/doxygen/master/structAVCodecContext.html#ab7bfeb9fa5840aac090e2b0bd0ef7589) property used in [libav.h#163](https://github.com/JuanKman94/readymedia-transcode/blob/transcode/libav.h#L161)

What's weird is that the program did compile 🤔  Will it cause a runtime error? I'm no expert. Have you streamed a lot with this build? If so, then I guess it's okay

Reply:
Interesting. I compiled it in Arch, also tested that it installs with AUR when I changed the source to my fork.

However, I have not streamed with this yet.

I see that warning too. I believe it's worth keeping the compile clean if it was clean earlier. Let me take a look.

Also, thanks for checking :)

Reply:
Argh... I realize this needs more changes.

* I am assigning ac multiple times here https://github.com/hirak99/readymedia-transcode/blob/transcode/dlnameta.c#L234, but freeing it up only once.
* As I look at it in git, I realized I'm using spaces but the code uses tabs.

Let me fix all these and check that it streams.

Reply:
It's working, streamed my old videos successfully.
My TV doesn't support audio streaming, so I couldn't test that.
And another weird thing, after I disconnect a connection, the report at localhost:8200 sometimes shows "-1 connections currently open". Possibly an edge condition from before, and not introduced by this change?

Reply:
And compiles without that warning anymore.

Reply:
I've been using this branch for more than a week now, no hiccups :+1: I'm going to merge :smile: 

BTW, I haven't played too much with the decoding scripts but I'm going to tweak them to see if I can stream some MKV videos to an old samsung smart tv. I'll check you branch from time to time to see if you've changed the transcode scripts as well.

Thanks for the contribution!